Shared/Tests.Shared/TestObjects.cs

```diff
@@ -209,6 +210,25 @@ public class Owner : RealmObject
     }
 
     [SuppressMessage("StyleCop.CSharp.MaintainabilityRules", "SA1402:FileMayOnlyContainASingleClass")]
+    public class Employee : RealmObject
+    {
+        public string Name { get; set; }
+
+        public IList<Employee> Reports { get; }
+
+        [Backlink(nameof(Employee.Reports))]
+        public IQueryable<Employee> Bosses { get; }
+
+        public Employee Boss
+        {
+            get
+            {
+                return Bosses.ToList().SingleOrDefault();
```

Comment:
I am quite opposed to using tests as documentation. Putting myself in users' shoes, if I get weird behavior, I wouldn't start browsing unit tests, I'll check the docs and if there's nothing in there, I'd open an issue.